apps/camera/test/unit/controllers/recording-timer_test.js

```diff
@@ -53,4 +64,33 @@ suite('controllers/recording-timer', function() {
       assert.isTrue(this.view.value.calledWith(0));
     });
   });
+
+  suite('Accessibility Tests', function() {
+
+    var app,
+      view,
+      controller;
+
+    setup(function() {
+      app = sinon.createStubInstance(this.App);
+      app.view = new this.RecordingTimerView();
+      view = app.view;
+      controller = new this.RecordingTimerController(app);
+    });
+
+    test('Explicitly test for RecordingTimer visibility', function() {
+      // App is in its initialized state
+      assert.equal(window.getComputedStyle(view.el, null).visibility, 'hidden');
```

Comment:
I think it's sufficient to just test for visible class being present/not present in these 3 cases.